charts/datarepo-api/values.yaml

```diff
@@ -72,3 +72,7 @@ existingApplicationSecretSecretKey: ""
 existingSecretRBS: ""
 ## The key in the existing secret that stores the RBS service account credentials
 existingRBSSecretKey: ""
+## The key in the existing secret that stores the azure synapse admin user name
```

Comment:
I'd move this to line 71 so that it's grouped with the azure key db (I'm assuming that's where these are going to live?)